dgea/plot_dgea.py

```diff
@@ -94,16 +94,30 @@ def plot_volcano():
         
         df_plot = deseq_results.copy()
         df_plot["-log10(padj)"] = -np.log10(df_plot["padj"])
+        min_value = 1e-100
+        df_plot["p-value"] = df_plot["padj"].apply(lambda x: x if x > min_value else f"less than {min_value}")
+
         df_plot["category"] = "Not significant"
         df_plot["gene"] = df_plot.index
-        df_plot.loc[(df_plot["log2FoldChange"] < -lfc) & (df_plot["padj"] < alpha), "category"] = "Downregulated"
-        df_plot.loc[(df_plot["log2FoldChange"] > lfc) & (df_plot["padj"] < alpha), "category"] = "Upregulated"
+        #df_plot.loc[(df_plot["log2FoldChange"] < -lfc) & (df_plot["padj"] < alpha), "category"] = "Downregulated"
+        #df_plot.loc[(df_plot["log2FoldChange"] > lfc) & (df_plot["padj"] < alpha), "category"] = "Upregulated"
```

Comment:
Remove this comments (if we still need them we can go back in version history)

Reply:
ok